docs/source/en/quick_tour.md

```diff
@@ -77,21 +77,22 @@ print(len(embedding[0]))
 ```
 
 #### OpenAI
+The server is availabe at : localhost:8080/v1/embeddings
 
 You can install it via pip as `pip install --upgrade openai`, and then run:
 
 ```python
 import os
 from openai import OpenAI
 
-client = OpenAI(base_url="http://localhost:8080/v1/embeddings")
+client = OpenAI(base_url="http://localhost:8080/v1", api_key= "fake")
 
 response = client.embeddings.create(
   model="tei",
   input="What is deep learning?"
 )
 
-print(response)
+print(response.data[0].embedding)
 ```
 
```

Comment:
Maybe also add `curl` example? Note that the closing backticks are missing another one, but GitHub won't let me add that and still be compliant with the suggestion format, so you'll need to add it afterwards 🤗 

```suggestion

Alternatively, you can also send the request with cURL as follows:

```bash
curl http://localhost/v1/embeddings \
  -H "Content-Type: application/json" \
  -d '{
    "input": "What is Deep Learning?",
    "model": "text-embeddings-inference",
    "encoding_format": "float"
  }'
``

```